src/app/cart/page.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+
+import React, { useEffect, useState } from "react";
+import { Box, Button, Typography } from "@mui/material";
+
+import ProductsList from "@/components/cart/ProductsList/ProductsList";
+import ModalConfirm from "@/components/cart/ModalConfirm/ModalConfirm";
+import TotalPrice from "@/components/cart/TotalPrice/TotalPrice";
+
+import styles from "./page.styles";
+import constants from "./page.constants";
+
+const Cart = () => {
+  const [cartItems, setCartItems] = useState<string | null>(null);
+  const [open, setOpen] = useState(false);
+
+  useEffect(() => setCartItems(localStorage.getItem("cart")), []);
+
+  const parsedCartItems = cartItems ? JSON.parse(cartItems) : [];
```

Comment:
Consider using some custom hook to work with local storage, this functionality could be repeated